README.md

```diff
@@ -102,18 +102,18 @@ Take a look at all the functions that are available [here](https://github.com/ar
 # Load text data
 
 Pre-processing text data is useful only if you have loaded data to process! Importing text data as strings in your code can be really simple if you have short texts contained in a local .txt, but it can quickly become difficult if you want to load a lot of texts, stored in multiple formats and divided in multiple files. Hopefully, you can use NLPretext's TextLoader class to easily import text data.
-Our TextLoader class makes use of Dask, so be sure to install the library if you want to use it, as mentioned above.
+Our TextLoader class can make use of dask, make sure to have the librairy installed if you want to use it.
```

Comment:
I think we should advice users that Dask is the library that is used by default, so if they don't want to use it, they should specify "use_dask=False". Otherwise they should make sure that the library is installed

Reply:
I don't fully agree with you on that. as you stated before, dask shouldn't be in the requirement. Thus the default behavior is not to use it

Reply:
I think Dask is what makes the TextLoader really useful, so even if we don't force users to use it, I do think it should be the default behavior. I don't think users will investigate a lot to see how they can optimize the way they load data, so the best way to do it in my opinion is using Dask by default, but warn them that it's not an obligation to use it.

As you handled the case where Dask is not installed by immediately switch to pandas with a warning, people that don't have Dask won't encounter any problem, just a warning message.